app/routes/api/$q/index.tsx

```diff
@@ -0,0 +1,20 @@
+import {
```

Comment:
I don't think this file should be a `.tsx`

Reply:
Yup this is a remains from when I was trying to debug it and see whether extension made a difference